hiro/cocoa/widget/hex-edit.cpp

```diff
@@ -5,43 +5,163 @@
 -(id) initWith:(hiro::mHexEdit&)hexEditReference {
   if(self = [super initWithFrame:NSMakeRect(0, 0, 0, 0)]) {
     hexEdit = &hexEditReference;
+    if (tableView = [[NSTableView alloc] initWithFrame:[self bounds]]) {
+      [tableView setDataSource:self];
+      [tableView setHeaderView:nil];
+      
+      // remove padding and stuff to fit all 16 columns
+      [tableView setIntercellSpacing:NSMakeSize(0, 0)];
+      [tableView setStyle:NSTableViewStylePlain];
+      NSTableColumn* addressCol = [[NSTableColumn alloc] initWithIdentifier:@"Address"];
```

Comment:
```suggestion
      NSTableColumn* addressCol = [[NSTableColumn alloc] initWithIdentifier:@"Address"];
      [tableView setUsesAlternatingRowBackgroundColors:true];
```

I'd suggest leveraging the built-in functionality for alternating row backgrounds, since it should help out with legibility for our data-dense table.